expr/seriesaggregators.go

```diff
@@ -36,6 +36,10 @@ func getCrossSeriesAggFunc(c string) crossSeriesAggFunc {
 		return crossSeriesStddev
 	case "rangeOf", "range":
 		return crossSeriesRange
+	case "last", "current":
+		return crossSeriesLast
+	case "count":
+		return crossSeriesCount
```

Comment:
good additions. they weren't documented in graphite either so i filed a pr for graphite docs.
https://github.com/graphite-project/graphite-web/pull/2451/files

Reply:
seems we are missing avg_zero

Reply:
added avg_zero!